tests/test_lsst.py

```diff
@@ -0,0 +1,928 @@
+from __future__ import with_statement
+import unittest
+import numpy as np
+import warnings
+import os
+import galsim
+from galsim.lsst import LsstCamera, LsstWCS, _nativeLonLatFromRaDec
+from galsim.celestial import CelestialCoord
+import lsst.afw.geom as afwGeom
+
+def haversine(long1, lat1, long2, lat2):
+    """
+    Return the angular distance between two points in radians
+
+    inputs
+    ------------
+    long1 is the longitude of point 1 in radians
+
+    lat1 is the latitude of point 1 in radians
+
+    long2 is the longitude of point 2 in radians
+
+    lat2 is the latitude of point 2 in radians
+
+    outputs
+    ------------
+    the angular separation between points 1 and 2 in radians
+
+    From http://en.wikipedia.org/wiki/Haversine_formula
+    """
+    t1 = np.sin(lat2/2.-lat1/2.)**2
+    t2 = np.cos(lat1)*np.cos(lat2)*np.sin(long2/2. - long1/2.)**2
+    return 2*np.arcsin(np.sqrt(t1 + t2))
```

Comment:
This is fine, although the tests would probably be a bit more self-documenting if they used the CelestialCoord `distance` function.  The term `haversine` is a bit jargonny compared to `distance`